XSLT/chattpubdctermstomods.xsl

```diff
@@ -114,6 +117,15 @@
         </xsl:choose>
     </xsl:template>
     
+    <!-- collection title -->
+    <xsl:template match='dcterms:isPartOf'>
+        <relatedItem displayLabel="Collection" type="host">
```

Comment:
@mlhale7 It's been so long since I've been involved or kept up with things that I can't remember how DPLA's aggregation stuff works.  Our mapping docs state they map the following to `Collection`:

```
<relatedItem displayLabel="Project"><titleInfo><title> AND <relatedItem displayLabel="collection"><titleInfo><title>
```

Does this need to match this more closely? I just don't remember what their tool chain looks like and how correct xpaths need to be.

Reply:
So, it's clear to me that they do pull in both for our data. See https://dp.la/item/54a7c476dab1c43d6bfd88dc0781e109 Honestly I'm not certain that we really want both (likely just @displayLabel="Collection"), but for the purposes of Chattanooga Public I'm confident that we'll get what we need.

Reply:
Cool, what i meant was i wasn't sure if it was okay that the `c` was lowercase and there was an extra attribute. Looking at this record, it seems like a similar pattern was followed.